src/jquery.mask.js

```diff
@@ -467,6 +472,10 @@
         return this.data('mask').getCleanVal();
     };
 
+    $.fn.maskedVal = function(val) {
```

Comment:
> Instead of creating a new public method wouldn't be better just returning p.getMasked() on the .mask public function?
It is ultimately your choice, but I wouldn't want to risk breaking compatibility if people use the chaining pattern when using the mask() public function.  I could, however, add a value option which returns p.getMasked instead of this when set.  Tell me which is your preferred way.

I'll edit this with a link to a jsFiddle or plunkr soon.